server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -90,26 +94,25 @@ module.exports = class ReportUploadRequestHandler {
     });
   }
 
-  isJSONFormat(line) {
+  isJSONFormat(line: string): boolean {
     return line[0] === '[';
   }
 
-  async loadData() {
-    const items = [];
-    const errors = [];
+  async loadData(): Promise<any[]> {
```

Comment:
Can you replace any with something more specific?

Reply:
yeah we can use string